themes/default/layouts/shortcodes/configure-gcp.html

```diff
@@ -13,10 +14,10 @@
     </div>
 </div>
 
-Next, Pulumi requires default application credentials to interact with your Google Cloud resources, so run `auth application-default login` command to obtain those credentials:
+To configure Pulumi to interact with your Google Cloud project, set it with the `config` command using the project's ID:
 
 <div class="highlight">
-    <pre class="chroma"><code class="language-bash" data-lang="bash"><span class="no-select">$ </span>gcloud auth application-default login</code></pre>
+    <pre class="chroma"><code class="language-bash" data-lang="bash"><span class="no-select">$ </span>pulumi config set gcp:project your-gcp-project-id</code></pre>
```

Comment:
We also support 3 environment variables so it doesn't need to be via config

Reply:
I go into this further below in this doc, but it's in this PR and waiting on publish:
https://github.com/pulumi/registry/pull/1369

Reply:
Added a snippet for this